copy/src/bin/main.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright © 2019-2020 The Radicle Foundation <hello@radicle.foundation>
+//
+// This file is part of radicle-link, distributed under the GPLv3 with Radicle
+// Linking Exception. For full terms see the included LICENSE file.
+
+use std::{
+    convert::TryFrom as _,
+    path::{Path, PathBuf},
+};
+
+use anyhow::anyhow;
+use argh::FromArgs;
+
+use librad::{
+    git::{
+        identities::{self, local},
+        storage::Storage,
+        Urn,
+    },
+    git_ext::{OneLevel, RefLike},
+    internal::canonical::Cstring,
+    keys::{PublicKey, SecretKey},
+    paths::Paths,
+    peer::PeerId,
+    signer::{BoxedSigner, SomeSigner},
+};
+use radicle_keystore::{
+    crypto::{self, Pwhash},
+    pinentry::Prompt,
+    FileStorage,
+    Keystore,
+};
+
+const SECRET_KEY_FILE: &str = "librad.key";
+
+/// Management of Radicle projects and their working-copies.
+///
+/// This tools allows you to create projects in your Radicle store and manage
+/// the remotes for their working copies.
+#[derive(Debug, FromArgs)]
+pub struct Args {
+    #[argh(subcommand)]
+    command: Command,
+}
+
+#[derive(Debug, FromArgs)]
+#[argh(subcommand)]
+pub enum Command {
+    Existing(Existing),
+    Fork(Fork),
+    New(New),
+    Update(Update),
+}
+
+/// 🆙 Update the remotes that exist in the include file for the given project
+#[derive(Debug, FromArgs)]
+#[argh(subcommand, name = "include-update")]
```

Comment:
Oh one annoying thing about `argh` is that we can't do something like `include update` as sub-sub-commands. Or at least not that I can see, so I created a ticket https://github.com/google/argh/issues/79

Reply:
I think you need to insert an extra indirection through a `struct`, instead of ~~doing the obvious thing~~ nesting `enum`s: https://github.com/google/argh/blob/master/tests/lib.rs#L612-L630

Reply:
arrrggghhhhhhhh, thanks for finding that. I'll try it out